onnxruntime_extensions/_cuops.py

```diff
@@ -388,6 +388,19 @@ def get_outputs(cls):
         ]
 
 
+class AudioDecoder(CustomOp):
+    @classmethod
+    def get_inputs(cls):
+        return [
+            cls.io_def('audio_stream', onnx_proto.TensorProto.UINT8, [1, None])
```

Comment:
not extremely important but at some point for all custom ops we should probably denote what the shapes represent for better readability, would help when modifying them in the future.

Reply:
Yes, the string of name is better.
1 was intentional since the operator now only support batch size = 1